src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -41,10 +56,18 @@ public function configureOptions(OptionsResolver $resolver)
                     return null;
                 }
 
-                return $this;
+                return new CallbackChoiceLoader(function () use ($options) {
+                    return self::getTimezones($options['regions']);
+                });
```

Comment:
I would add `$regions = $options['regions']` and pass that only variable to the `use` statement.